src/main/resources/jupyter/init-actions.sh

```diff
@@ -82,10 +83,15 @@ if [[ "${ROLE}" == 'Master' ]]; then
 
     docker-compose -f /etc/cluster-docker-compose.yaml up -d
 
+    # The Dataproc-provided spark-defaults.conf is compose'd into the Docker; append our custom settings to the end of it
+    docker exec -d ${JUPYTER_SERVER_NAME} /etc/construct-spark-defaults.sh
```

Comment:
docker 1.6 (i.e. `apt-get install docker.io`) doesn't run `docker exec` as root, so this fails because it's touching `/etc`